app/build.gradle

```diff
@@ -24,19 +24,18 @@ static def gitSha() {
 android {
     compileSdk 34
     defaultConfig {
-        applicationId "org.gnucash.android"
-        testApplicationId 'org.gnucash.android.test'
+        applicationId "org.gnucash.pocket"
         minSdkVersion 21
         targetSdkVersion 33
         versionCode versionMajor * 10000 + versionMinor * 1000 + versionPatch * 100 + versionBuild
         versionName "${versionMajor}.${versionMinor}.${versionPatch}"
         resValue "string", "app_version_name", "${versionName}"
         resValue "string", "app_minor_version", "${versionMinor}"
+        resValue "string", "app_playstore_url", "market://details?id=${applicationId}"
         buildConfigField "boolean", "CAN_REQUEST_RATING", "false"
         buildConfigField "String", "BUILD_TIME", "\"${buildTime()}\""
 
         testInstrumentationRunner "org.gnucash.android.test.ui.util.GnucashAndroidTestRunner"
```

Comment:
Shouldn't it be  `org.gnucash.pocket.…` and `….GnuCashPocketTestRunner`? (Also in line 157)

Reply:
Also line 49 `GnuCashPocket_v${variant.versionName}.apk` ?

Reply:
whoever modifies his file to `org.gnucash.pocket.XXX` will get the relevant applicationId.
no need to change file names.
no need to change package names.